.rubocop.yml

```diff
@@ -0,0 +1,14 @@
+AllCops:
+  NewCops: enable
+  Exclude:
+    - db/**/*
+    - bin/**/*
+    - node_modules/**/*
+    - spec/*.rb
```

Comment:
I think listing spec files makes sense too. Is there any reason it needs to be excluded?